src/test/regress/sql/multi_subquery_behavioral_analytics.sql

```diff
@@ -1285,6 +1285,59 @@ SELECT * FROM run_command_on_workers('DROP FUNCTION array_index(ANYARRAY, ANYELE
 ORDER BY 1,2;
 DROP FUNCTION array_index(ANYARRAY, ANYELEMENT);
 
+-- a not supported query due to constant range table entry
+SELECT count(*) as subquery_count
+FROM (
+  SELECT 
+      user_id
+    FROM
+    users_table
+  WHERE
+    (value_1 = '5' OR value_1 = '13')
+  GROUP BY user_id                                                                          
```

Comment:
I think we don't have a formal one, but having symmetry and consistent spacing makes it easier to read and understand queries. It doesn't need to be strict as much as our coding conventions.